addons/crm_livechat/models/discuss_channel.py

```diff
@@ -2,13 +2,24 @@
 
 from markupsafe import Markup
 
-from odoo import models, _
+from odoo import fields, models, _
 from odoo.tools import html2plaintext
 
 
 class DiscussChannel(models.Model):
     _inherit = 'discuss.channel'
 
+    crm_lead_ids = fields.One2many("crm.lead", "channel_id")
```

Comment:
This field should probably be restricted to sales people
```suggestion
    crm_lead_ids = fields.One2many("crm.lead", "channel_id", groups="sales_team.group_sale_salesman")
```
Same below.